backend/app/settings.py

```diff
@@ -0,0 +1,32 @@
+import secrets
+
+from pydantic import BaseSettings
+
+
+class Settings(BaseSettings):
+    # The base URL to the login portal
+    OIDC_PROVIDER: str = "https://login.sandbox.ioxio-dataspace.com"
+    OIDC_SCOPES: str = "openid"
+    OIDC_ACR_VALUES: str = "fake-auth"
+
+    # The client ID and client secret for the app you created in the developer portal
+    OIDC_CLIENT_ID: str = ""
+    OIDC_CLIENT_SECRET: str = ""
+
+    # Secret used to protect the session
+    SESSION_SECRET: str = secrets.token_urlsafe()
```

Comment:
Add a comment here that this is currently generated at execution time meaning it changes. Real deployments should not do this.